logic/src/commonTest/kotlin/com/wire/kalium/logic/data/user/UserRepositoryTest.kt

```diff
@@ -412,87 +366,6 @@ class UserRepositoryTest {
         }
     }
 
-    @Test
-    fun givenUpdateEmailSuccess_whenChangingEmail_thenSuccessIsReturned() = runTest {
-        val (arrangement, userRepository) = Arrangement()
-            .withRemoteUpdateEmail(NetworkResponse.Success(true, mapOf(), 200))
-            .arrange()
-
-        val result = userRepository.updateSelfEmail("newEmail")
-
-        with(result) {
-            shouldSucceed()
-            verify(arrangement.selfApi)
-                .suspendFunction(arrangement.selfApi::updateEmailAddress)
-                .with(eq("newEmail"))
-                .wasInvoked(exactly = once)
-        }
-    }
-
-    @Test
-    fun givenUpdateEmailFailure_whenChangingEmail_thenFailureIsReturned() = runTest {
-        val (arrangement, userRepository) = Arrangement()
-            .withRemoteUpdateEmail(NetworkResponse.Error(KaliumException.GenericError(IOException())))
-            .arrange()
-
-        val result = userRepository.updateSelfEmail("newEmail")
-
-        with(result) {
-            shouldFail()
-            verify(arrangement.selfApi)
-                .suspendFunction(arrangement.selfApi::updateEmailAddress)
-                .with(eq("newEmail"))
-                .wasInvoked(exactly = once)
-        }
-    }
-
-    @Test
-    fun givenThereAreUsersWithoutMetadata_whenSyncingUsers_thenShouldUpdateThem() = runTest {
```

Comment:
Isn't this test valid anymore? I see it was removed and doesn't seem like it was moved to `AccountRepositoryTest`.

Reply:
i think it was deleted when solving merge conflict with dev, i will add it